tox.ini

```diff
@@ -1,6 +1,34 @@
 [tox]
-envlist = py27,py33,py34,py35
+envlist = lint,py27,py33,py34,py35
```

Comment:
These versions don't line up with our requirements. 

IMO something to fix separately, lmk if you'd like me to file an issue.

Reply:
That's handled by #660.